docs/docsite/rst/getting_started/yaml/inventory_example_vms.yaml

```diff
@@ -1,8 +1,8 @@
-virtualmachines:
+myhosts:
```

Comment:
This is a good point. It probably would make things simpler. I'll give it a go, thanks.

Reply:
oh no, not the INI vs YML wars lol! We actually ended up adding both INI and YAML in the [inventory guide](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html) because it really is a ...war!

Reply:
lol, i dont think its a war, i think both INI and YML proponents know where the strengths of  both syntax variants are and they want to use it accordingly (with all due respect) :-)